
github.com/ofiwg/libfabric/pull/10836
github.com/ofiwg/libfabric/commit/bd420df

From bd420dfe3a563ba94d199403d8fb2ec3cfe3053a Mon Sep 17 00:00:00 2001
From: Orion Poplawski <orion@nwra.com>
Date: Sat, 1 Mar 2025 20:00:22 -0700
Subject: [PATCH] Fix some test function prototypes


adapted to opensuse's source tree

--- fabtests/ubertest/config.c
+++ fabtests/ubertest/config.c
@@ -712,7 +712,7 @@ int fts_info_is_valid(void)
 			return 0;
 	}
 	if (test_info.test_class & (FI_MSG | FI_TAGGED) &&
-	    !ft_check_rx_completion(test_info) &&
+	    !ft_check_rx_completion() &&
 	    !ft_use_comp_cntr(test_info.comp_type))
 		return 0;
 	if (test_info.test_type == FT_TEST_UNIT &&
--- fabtests/ubertest/fabtest.h
+++ fabtests/ubertest/fabtest.h
@@ -362,7 +362,7 @@ int ft_check_rx_completion();
 int ft_check_tx_completion();
 
 int ft_send_sync_msg();
-int ft_recv_n_msg();
+int ft_recv_n_msg(int n);
 int ft_recv_msg();
 int ft_send_msg();
 int ft_send_dgram();
